.github/workflows/build.yaml

```diff
@@ -39,8 +40,6 @@ jobs:
       uses: actions/setup-go@v2
       with:
         go-version: 1.15.2
-    - name: Create cache directory as pre-commit would do
```

Comment:
Because we don't need it anymore.
The code was modified to not assume presence of "pre-commit" directories, instead it creates them if needed (`os.makedirs` in this PR)